flow/Knobs.cpp

```diff
@@ -47,6 +47,7 @@ FlowKnobs::FlowKnobs(bool randomize, bool isSimulated) {
 	init( SLOWTASK_PROFILING_LOG_BACKOFF,                      2.0 );
 
 	init( RANDOMSEED_RETRY_LIMIT,                                4 );
+	init( FAST_ALLOC_LOGGING_CHANCE,          (128<<10)/10000000.0 ); //one backtrace every 10MB
```

Comment:
Should we make the magazine size a knob? Or at the very least, we could change this knob to 10e6 (or 10e-6) and then move the magazine size constant to FastAlloc.cpp in order to limit its spread.